3_parameter_efficient_finetuning/README.md

```diff
@@ -11,20 +11,29 @@ Traditional fine-tuning updates all model parameters during training, which beco
 
 ## Available Methods
 
-### [LoRA (Low-Rank Adaptation)](./lora_adapters.md)
+In this module, we will cover two popular PEFT methods:
 
-LoRA has emerged as the most widely adopted PEFT method, offering an elegant solution to efficient model adaptation. Instead of modifying the entire model, LoRA injects trainable rank decomposition matrices into the model's attention layers. This approach typically reduces trainable parameters by about 90% while maintaining comparable performance to full fine-tuning.
+### 1️⃣ LoRA (Low-Rank Adaptation)
 
-### [Prompt Tuning](./prompt_tuning.md)
+LoRA has emerged as the most widely adopted PEFT method, offering an elegant solution to efficient model adaptation. Instead of modifying the entire model, LoRA injects trainable matrices into the model's attention layers. This approach typically reduces trainable parameters by about 90% while maintaining comparable performance to full fine-tuning. We will explore LoRA in the [LoRA (Low-Rank Adaptation)](./lora_adapters.md) section.
```

Comment:
```suggestion
LoRA has emerged as the most widely adopted PEFT method, offering an elegant solution to efficient model adaptation. Instead of modifying the entire model, **LoRA injects trainable matrices into the model's attention layers.** This approach typically reduces trainable parameters by about 90% while maintaining comparable performance to full fine-tuning. We will explore LoRA in the [LoRA (Low-Rank Adaptation)](./lora_adapters.md) section.
```